addons/knobs/src/components/Panel.js

```diff
@@ -196,9 +196,9 @@ export default class KnobPanel extends PureComponent {
       );
     }
 
-    const entries = Object.entries(groups);
     // Always sort 'Other' (ungrouped) tab last without changing the remaining tabs
-    entries.sort((a, b) => (a[0] === 'Other' ? 1 : 0)); // eslint-disable-line no-unused-vars
+    const entries = Object.entries(groups).filter(entry => entry[0] !== 'Other');
```

Comment:
Minor optimization:
```
const unsortedEntries = Object.entries(groups)
const entries = unsortedEntries.filter(...).push(...)
```

Also we should be using the const `DEFAULT_GROUP_ID` defined at the top of the file?

Reply:
Excellent point about the const, didn't even realise it was there since it wasn't used in the previous sorting implementation. Pushing an update right now.

Reply:
Turns out that doesn't work @shilman. Push returns the length of the array after adding the item back. My bad for not testing it properly before pushing the update.